ttnctl/cmd/applications_pf.go

```diff
@@ -33,7 +33,7 @@ function Decoder(bytes) {
 
 		appID := util.GetAppID(ctx)
 
-		conn, manager := util.GetHandlerManager(ctx)
+		conn, manager := util.GetHandlerManager(ctx, appID)
 		defer conn.Close()
 
 		app, err := manager.GetApplication(appID)
```

Comment:
Why do we need to pass `appID` to `manager.GetApplication` when `manager` is already scoped to this `AppID`? (same for other `ttnctl` cmds)